### ISSUE TITLE
Question: How to get current user granted permission scopes?

### ISSUE THREAD

**Issue by Ralms** [AzureAD/microsoft-authentication-library-for-js#854]
## I'm submitting a...

<pre><code>
[ ] Regression (a behavior that used to work and stopped working in a new release)
[ ] Bug report  
[ ] Performance issue
[ ] Feature request
[x ] Documentation issue or request
[ ] Other... Please describe:
</code></pre>

## Browser:
- [ ] Chrome version XX
- [ ] Firefox version XX
- [ ] IE version XX
- [ ] Edge version XX
- [ ] Safari version XX
- [x] Opera 62.0.3331.72

## Library version

<pre><code>
Library version: 1.0.2
</code></pre>

## Question
Currently the "Account" class doesn't contain permission scopes property.
How can I get the granted permission scopes for the current user?
Thanks.

**Comment by pkanher617** [comment]
@Ralms The granted scopes are returned in the AuthResponse object that is returned when an acquireToken or login call has been completed. 

```javascript

const accessTokenRequest = {
    scopes: ["user.read"]
}

userAgentApplication.acquireTokenSilent(accessTokenRequest).then(function(accessTokenResponse) {
    // Acquire token silent success
    // call API with token
    let accessToken = accessTokenResponse.accessToken;
    let scopes = accessTokenResponse.scopes;
}).catch(function (error) {
    //Acquire token silent failure, send an interactive request.
    if (error.errorMessage.indexOf("interaction_required") !== -1) {
        userAgentApplication.acquireTokenPopup(accessTokenRequest).then(function(accessTokenResponse) {
            // Acquire token interactive success
            let scopes = accessTokenResponse.scopes;
        }).catch(function(error) {
            // Acquire token interactive failure
            console.log(error);
        });
    }
    console.log(error);
});
```

Hope this answers your question! Please reopen this issue if it isn't working as expected.

**Comment by Ralms** [comment]
Hi, 
Yes but what I'm looking was for a way for me to check those scopes before requesting the AccessToken.

Is there a way, like we have getAccount() for the user, but to get all Scopes that the user or Admin have granted to my application?

**Comment by Ralms** [comment]
And I don't seem to have permission to reopen issues.

**Comment by pkanher617** [comment]
Leaving this closed since it is related to #853.
